Autotools build fails

Current master (5a3a0b64534bcda33d385a270e1493a8319d28f8) fails to build using the Die GNU Autotools.

JFreegman and System_Error over on IRC confirm this too.


```
$ make
make  all-recursive
make[1]: Entering directory '/home/nurupo/git/c-toxcore'
Making all in build
make[2]: Entering directory '/home/nurupo/git/c-toxcore/build'
  CC       ../testing/DHT_test-DHT_test.o
  CC       ../testing/libmisc_tools_la-misc_tools.lo
  CCLD     libmisc_tools.la
  CC       ../toxcore/events/libtoxcore_la-conference_connected.lo
  CC       ../toxcore/events/libtoxcore_la-conference_invite.lo
  CC       ../toxcore/events/libtoxcore_la-conference_message.lo
  CC       ../toxcore/events/libtoxcore_la-conference_peer_list_changed.lo
  CC       ../toxcore/events/libtoxcore_la-conference_peer_name.lo
  CC       ../toxcore/events/libtoxcore_la-conference_title.lo
  CC       ../toxcore/events/libtoxcore_la-file_chunk_request.lo
  CC       ../toxcore/events/libtoxcore_la-file_recv.lo
  CC       ../toxcore/events/libtoxcore_la-file_recv_chunk.lo
  CC       ../toxcore/events/libtoxcore_la-file_recv_control.lo
  CC       ../toxcore/events/libtoxcore_la-friend_connection_status.lo
  CC       ../toxcore/events/libtoxcore_la-friend_lossless_packet.lo
  CC       ../toxcore/events/libtoxcore_la-friend_lossy_packet.lo
  CC       ../toxcore/events/libtoxcore_la-friend_message.lo
  CC       ../toxcore/events/libtoxcore_la-friend_name.lo
  CC       ../toxcore/events/libtoxcore_la-friend_read_receipt.lo
  CC       ../toxcore/events/libtoxcore_la-friend_request.lo
  CC       ../toxcore/events/libtoxcore_la-friend_status.lo
  CC       ../toxcore/events/libtoxcore_la-friend_status_message.lo
  CC       ../toxcore/events/libtoxcore_la-friend_typing.lo
  CC       ../toxcore/events/libtoxcore_la-events_alloc.lo
  CC       ../toxcore/events/libtoxcore_la-self_connection_status.lo
  CC       ../toxcore/libtoxcore_la-DHT.lo
  CC       ../toxcore/libtoxcore_la-mono_time.lo
  CC       ../toxcore/libtoxcore_la-network.lo
  CC       ../toxcore/libtoxcore_la-crypto_core.lo
  CC       ../toxcore/libtoxcore_la-ping_array.lo
  CC       ../toxcore/libtoxcore_la-net_crypto.lo
  CC       ../toxcore/libtoxcore_la-friend_requests.lo
  CC       ../toxcore/libtoxcore_la-LAN_discovery.lo
  CC       ../toxcore/libtoxcore_la-friend_connection.lo
  CC       ../toxcore/libtoxcore_la-Messenger.lo
  CC       ../toxcore/libtoxcore_la-ping.lo
  CC       ../toxcore/libtoxcore_la-state.lo
  CC       ../toxcore/libtoxcore_la-tox.lo
  CC       ../toxcore/libtoxcore_la-tox_dispatch.lo
  CC       ../toxcore/libtoxcore_la-tox_events.lo
  CC       ../toxcore/libtoxcore_la-tox_unpack.lo
  CC       ../toxcore/libtoxcore_la-tox_api.lo
  CC       ../toxcore/libtoxcore_la-util.lo
  CC       ../toxcore/libtoxcore_la-group.lo
  CC       ../toxcore/libtoxcore_la-onion.lo
  CC       ../toxcore/libtoxcore_la-logger.lo
  CC       ../toxcore/libtoxcore_la-onion_announce.lo
  CC       ../toxcore/libtoxcore_la-onion_client.lo
  CC       ../toxcore/libtoxcore_la-TCP_client.lo
  CC       ../toxcore/libtoxcore_la-TCP_common.lo
  CC       ../toxcore/libtoxcore_la-TCP_server.lo
  CC       ../toxcore/libtoxcore_la-TCP_connection.lo
  CC       ../toxcore/libtoxcore_la-list.lo
  CCLD     libtoxcore.la
/usr/bin/ld: ../toxcore/events/.libs/libtoxcore_la-friend_message.o: in function `tox_event_friend_message_get_friend_number':
/home/nurupo/git/c-toxcore/build/../toxcore/events/friend_message.c:50: multiple definition of `tox_event_friend_message_get_friend_number'; ../toxcore/events/.libs/libtoxcore_la-friend_message.o:/home/nurupo/git/c-toxcore/build/../toxcore/events/friend_message.c:50: first defined here
/usr/bin/ld: ../toxcore/events/.libs/libtoxcore_la-friend_message.o: in function `tox_event_friend_message_get_type':
/home/nurupo/git/c-toxcore/build/../toxcore/events/friend_message.c:61: multiple definition of `tox_event_friend_message_get_type'; ../toxcore/events/.libs/libtoxcore_la-friend_message.o:/home/nurupo/git/c-toxcore/build/../toxcore/events/friend_message.c:61: first defined here
/usr/bin/ld: ../toxcore/events/.libs/libtoxcore_la-friend_message.o: in function `tox_event_friend_message_get_message_length':
/home/nurupo/git/c-toxcore/build/../toxcore/events/friend_message.c:88: multiple definition of `tox_event_friend_message_get_message_length'; ../toxcore/events/.libs/libtoxcore_la-friend_message.o:/home/nurupo/git/c-toxcore/build/../toxcore/events/friend_message.c:88: first defined here
/usr/bin/ld: ../toxcore/events/.libs/libtoxcore_la-friend_message.o: in function `tox_event_friend_message_get_message':
/home/nurupo/git/c-toxcore/build/../toxcore/events/friend_message.c:93: multiple definition of `tox_event_friend_message_get_message'; ../toxcore/events/.libs/libtoxcore_la-friend_message.o:/home/nurupo/git/c-toxcore/build/../toxcore/events/friend_message.c:93: first defined here
/usr/bin/ld: ../toxcore/events/.libs/libtoxcore_la-friend_message.o: in function `tox_events_clear_friend_message':
/home/nurupo/git/c-toxcore/build/../toxcore/events/friend_message.c:157: multiple definition of `tox_events_clear_friend_message'; ../toxcore/events/.libs/libtoxcore_la-friend_message.o:/home/nurupo/git/c-toxcore/build/../toxcore/events/friend_message.c:157: first defined here
/usr/bin/ld: ../toxcore/events/.libs/libtoxcore_la-friend_message.o: in function `tox_events_get_friend_message_size':
/home/nurupo/git/c-toxcore/build/../toxcore/events/friend_message.c:173: multiple definition of `tox_events_get_friend_message_size'; ../toxcore/events/.libs/libtoxcore_la-friend_message.o:/home/nurupo/git/c-toxcore/build/../toxcore/events/friend_message.c:173: first defined here
/usr/bin/ld: ../toxcore/events/.libs/libtoxcore_la-friend_message.o: in function `tox_events_get_friend_message':
/home/nurupo/git/c-toxcore/build/../toxcore/events/friend_message.c:181: multiple definition of `tox_events_get_friend_message'; ../toxcore/events/.libs/libtoxcore_la-friend_message.o:/home/nurupo/git/c-toxcore/build/../toxcore/events/friend_message.c:181: first defined here
/usr/bin/ld: ../toxcore/events/.libs/libtoxcore_la-friend_message.o: in function `tox_events_pack_friend_message':
/home/nurupo/git/c-toxcore/build/../toxcore/events/friend_message.c:188: multiple definition of `tox_events_pack_friend_message'; ../toxcore/events/.libs/libtoxcore_la-friend_message.o:/home/nurupo/git/c-toxcore/build/../toxcore/events/friend_message.c:188: first defined here
/usr/bin/ld: ../toxcore/events/.libs/libtoxcore_la-friend_message.o: in function `tox_events_unpack_friend_message':
/home/nurupo/git/c-toxcore/build/../toxcore/events/friend_message.c:200: multiple definition of `tox_events_unpack_friend_message'; ../toxcore/events/.libs/libtoxcore_la-friend_message.o:/home/nurupo/git/c-toxcore/build/../toxcore/events/friend_message.c:200: first defined here
/usr/bin/ld: ../toxcore/events/.libs/libtoxcore_la-friend_message.o: in function `tox_events_handle_friend_message':
/home/nurupo/git/c-toxcore/build/../toxcore/events/friend_message.c:229: multiple definition of `tox_events_handle_friend_message'; ../toxcore/events/.libs/libtoxcore_la-friend_message.o:/home/nurupo/git/c-toxcore/build/../toxcore/events/friend_message.c:229: first defined here
collect2: error: ld returned 1 exit status
make[2]: *** [Makefile:2185: libtoxcore.la] Error 1
make[2]: Leaving directory '/home/nurupo/git/c-toxcore/build'
make[1]: *** [Makefile:493: all-recursive] Error 1
make[1]: Leaving directory '/home/nurupo/git/c-toxcore'
make: *** [Makefile:413: all] Error 2
```

--- a/toxcore/Makefile.inc
+++ b/toxcore/Makefile.inc
@@ -22,7 +22,6 @@ libtoxcore_la_SOURCES = ../toxcore/attributes.h \
                         ../toxcore/events/friend_lossless_packet.c \
                         ../toxcore/events/friend_lossy_packet.c \
                         ../toxcore/events/friend_message.c \
-                        ../toxcore/events/friend_message.c \
                         ../toxcore/events/friend_name.c \
                         ../toxcore/events/friend_read_receipt.c \
                         ../toxcore/events/friend_request.c \

